backend/slack_commands/execution/session_executor.py

```
@@ -0,0 +1,164 @@
+                self._post_to_slack(response_url, text, success=True)
+            else:
+                self._post_to_slack(
+                    response_url,
+                    f":x: Session ended with status: *{status}*\n"
+                    f"_Session ID: `{session_id}`_",
+                )
+
+        except requests.HTTPError as e:
+            status_code = e.response.status_code if e.response is not None else "?"
+            body = self._extract_error_body(e)
+            logger.error("Session HTTP error: %s %s", status_code, body, exc_info=True)
```

Ruff (0.15.15)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)